python/plugins/processing/algs/qgis/Climb.py

```diff
@@ -145,15 +145,15 @@ def processAlgorithm(self, parameters, context, feedback):
 
         # Skip fields with names that are equal to the generated ones
         for field in source.fields():
-            if str(field.name()) == str(self.CLIMBATTRIBUTE):
-                feedback.pushInfo("Warning: existing " +
+            if field.name() == self.CLIMBATTRIBUTE:
+                feedback.pushWarning(self.tr(
                                   str(self.CLIMBATTRIBUTE) +
-                                  " attribute found and removed")
+                                  ' attribute found and removed'))
```

Comment:
Use formatting to make translation more flexible

`'{attr_name} attribute found and removed'.format(attr_name=self.CLIMBATTRIBUTE)`

Reply:
done, thanks for the hint